### DISCUSSION TITLE
Responsive props not working as expected

### DISCUSSION THREAD

**Discussion by baba43** [mantinedev/mantine#4883] [Q&A] [answered]
Hello all, I am trying out Mantine 7 (latest release) and want my `Container` to have no padding on smaller devices.

I thought that I could use the `p` prop with according breakpoint settings:

```
<Container
  p={{
    xl: "4em",
    lg: "3em",
    md: "2em",
    sm: "1em",
    xs: 0,
  }}
>
  <Card>...</Card>
</Container>
```

However, this is not working, as `xs` is using the media query `min-width: 36em`, causing the styles to reset to default when the screen size is below `36em`. What would be needed is a way to set the _default_ padding. 

Maybe I got something wrong or maybe I am doing it the wrong way overal, so I am glad for any advice. 

/edit
The same issue applies to different components. Here I wanted to set a lower header height for mobile and the height becomes 0 when the device size is below `36em`:

```
<Box
  h={{
    sm: 300,
    xs: "10em",
  }}
/>
```

I have the same issue when specifying `fz` (font-size) based on breakpoints.

**Comment by baba43** [comment]
I somehow _fixed_ the problem for me by adding a new breakpoint and use this to specify values for lower sizes.

```
export const theme = createTheme({
  breakpoints: {
    xss: "1em",
  },
});
```

However, I am not sure if there will be any downsides to this approach so my question remains open.

**Reply by henrymcl** [reply]
It seems that the Grid.Col component doesn't consume custom breakpoints for the span values.

**Comment by rtivital** [comment, marked as answer]
To specify values lower than `xs`, use `base` – https://mantine.dev/styles/style-props/#responsive-styles

**Reply by baba43** [reply]
Ah, thank you so much.
